central/policy/service/service_impl.go

```diff
@@ -314,12 +314,18 @@ func (s *serviceImpl) PatchPolicy(ctx context.Context, request *v1.PatchPolicyRe
 	return s.PutPolicy(ctx, policy)
 }
 
-// DeletePolicy deletes an policy from the system.
+// DeletePolicy deletes a policy from the system.
 func (s *serviceImpl) DeletePolicy(ctx context.Context, request *v1.ResourceByID) (*v1.Empty, error) {
 	if request.GetId() == "" {
 		return nil, errors.Wrap(errox.InvalidArgs, "A policy id must be specified to delete a Policy")
 	}
 
+	// Note: default policies cannot be deleted, only disabled
+	policy, _, _ := s.policies.GetPolicy(ctx, request.GetId())
+	if policy.IsDefault == true {
+		return nil, errors.Wrap(errox.NotImplemented, "A default Policy cannot be deleted. (You can disable a default policy, but not delete it.)")
```

Comment:
s/Policy/policy. "A default policy can be disabled, but not deleted."

Reply:
Also errox.NotImplemented seems wrong. We have intentionally implemented this :-). So errorx.InvalidArgs ? Since it is an "invalid" policy as far as delete is concerned?